README_dev.md

```diff
@@ -4,7 +4,8 @@
 2. download and install [Node JS](https://nodejs.org/en/download/)
 3. install the npm packages `typescript` and `cpx` **globally**
 4. Optional: install the VS Code extension [Tasks](https://marketplace.visualstudio.com/items?itemName=actboy168.tasks)
-5. run `npm i`
+5. Optional: restarting the Connector using the VSCode task `More Tasks > Restart` requires the tool socat. Make sure to install it on your system. This can be done for example using `apt-get install socat` on Ubuntu and `brew install socat` using Homebrew.
```

Comment:
Why optional? Why "make sure to install it ..." when it is optional?

Reply:
I think it is optional as it is only required for manual restart. By default, the restart will be handled by nodemon

Reply:
I mean, partly true: nodemon watches the files and restarts the server on changes. But sometimes (e.g. when nodemon fails to startup the server) we need to have a manual way to do it.

But yeah, it's very optional b/c it's only needed soooooometimes :D